src/app/legacy/containers/RelatedTopics/index.jsx

```diff
@@ -46,9 +46,12 @@ const RelatedTopics = ({
   const topicsPath = pathOr('topics', ['topicsPath'], translations);
 
   const getTopicPageUrl = id => {
+    const isPublicService = ['news', 'cymrufyw', 'naidheachdan'];
```

Comment:
Just curious if this scenario can happen on the apps? Would they get served .amp versions of the pages _within_ the apps?

Reply:
I think that within the News app, it would route to a Webcore app page, rather than a Simorgh one?

This will ensure that for the amp article pages that we use the correct URL for the related topics - bbc.com / bbc.co.uk.